chef/automate
# changes for 5414 & 5415

### :nut_and_bolt: Description: What code changed, and why?
Data feed page changed 
1. Disclaimer text added
2. New gif added to empty data-feed page
3. Create  data feed button changed to New Integration



### :chains: Related Resources
- https://github.com/chef/automate/issues/5414
- https://github.com/chef/automate/issues/5415

### :+1: Definition of Done
On Data feed page
1. 
![image](https://user-images.githubusercontent.com/13744874/130778126-ff5f9e6e-c773-4987-ab58-a0fc3bc233e4.png)
changed to 
![image](https://user-images.githubusercontent.com/13744874/130778189-061b2eb3-8add-4e12-bb2d-650d241ff51c.png)
2. Added a button to add new Integration.

Automate data-feed page should look like the image below.
<img width="1779" alt="Screenshot 2021-08-18 at 3 41 38 PM" src="https://user-images.githubusercontent.com/13744874/129888532-2143d31e-2002-4466-bf02-e80c112e7871.png">

### :athletic_shoe: How to Build and Test the Change
1. Inside the hab studio
```
 rebuild component/automate-ui 
```
2. Once UI is up:
 Go to settings > click feat on tab to enable feature flag (on Chef Automate Data Feed option) and reload page - Data feed option should come below Notification tab
3. Click on data-feed 
4. Empty data-feed should look like the image.
<img width="1787" alt="Screenshot 2021-08-25 at 4 12 52 PM" src="https://user-images.githubusercontent.com/13744874/130778662-aa1d35bc-0000-49f8-9aa5-2dcf96e82b13.png">

To test:
1. Go to e2e/cypress folder in automate
2. run npm run cypress:local

### :white_check_mark: Checklist

**All PRs** must tick these:


With occasional exceptions, all PRs **from Progress employees** must tick these:


All PRs **from Progress employees** should tick these if appropriate:


*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
<img width="1787" alt="Screenshot 2021-08-25 at 4 12 52 PM" src="https://user-images.githubusercontent.com/13744874/130781935-7c1e6850-c680-4943-80e6-d5a1394efb07.png">
